standalone_self_upgrade.cf.in

```diff
@@ -155,8 +155,10 @@ bundle agent cfengine_software
       "$(sys.statedir)/MPF/."
         create => "true";
 
+      "$(sys.statedir)/MPF/desired-cfengine-package-version.json"
+        create => "true";
+
       "$(sys.statedir)/MPF/desired-cfengine-package-version.json" -> { "ENT-3592", "ENT-3937" }
-        create => "true",
         edit_line => cfengine_software_version_data;
```

Comment:
Isn't this a bug? I'd expect this change to make no difference.

Reply:
I must have patched this upstream already. It's true that there should be no functional difference when separated into its own promise or not. Best practice does dictate to separate into smallest parts so I am inclined to leave this further separation in place.